al2.pkr.hcl

```diff
@@ -29,14 +38,8 @@ source "amazon-ebs" "al2" {
   }
   ssh_interface = "public_ip"
   ssh_username  = "ec2-user"
-  tags = {
-    os_version          = "Amazon Linux 2"
-    source_image_name   = "{{ .SourceAMIName }}"
-    ecs_runtime_version = "Docker version ${var.docker_version}"
-    ecs_agent_version   = "${var.ecs_agent_version}"
-    ami_type            = "al2"
-    ami_version         = "2.0.${var.ami_version_al2}"
-  }
+  tags = "${local.merged_tags}"
```

Comment:
You'll likely need to run `./packer fmt .` in `/amazon-ecs-ami/` directory to get pull request static checks to pass.